### ISSUE TITLE
Auto complete

### ISSUE THREAD

**Issue by reblevins** [atom-ide-community/atom-ide-vue#35]
Hi,
First, thanks for this atom plugin, it's great.

I recently upgraded to version 1.4.0 and auto-complete is not working as expected.

Normally, when I begin typing, it displays a list of possibilities, I pick the one I want, hit enter or tab and it completes it for me. That part is working fine, however for some reason it is deleting the character right in front of where the cursor started.

Example:
I start typing:
`this.someProp`, `someProperty` shows in the list, I select it, then hit enter and it inserts `someProperty`, but removes the period.
So now it becomes `thissomeProperty` instead of what is should be is `this.someProperty` (notice the `.` is missing in the first one).

I'm fairly certain it is die to this package because when I disable this package, auto-complete works as expected.

Thanks again.

**Comment by phromo** [comment]
Also just discovered this, I'm SO thrilled and happy that atom-ide-community are updating the atom-ide-ui package.

As reblevins writes: running completion on myobject.thing (ctrl-space, select item in list) makes the editor end up with "myobjectthing" (without dot). Tested in .vue and .ts files, using Atom 1.52.0 and atom-ide-vue 1.4.0, autocomplete-plus 2.42.3.

If someone can point me in the right direction maybe I can help out as well. Thanks!

**Comment by aminya** [comment]
@UziTech  What is this error?
![image](https://user-images.githubusercontent.com/16418197/101959596-3942a180-3bcb-11eb-9fb5-b497847d5320.png)

**Comment by UziTech** [comment]
It looks like the language server threw the error `"Unhandled method textDocument/documentSymbol"` without more info I couldn't tell you more.

**Comment by aminya** [comment]
- install this package and `language-vue`
- open the `.vue` file in the spec folder. 
- open dev tools and see the error

**Comment by UziTech** [comment]
Looks like this might be the issue https://github.com/vuejs/vetur/issues/2387

**Comment by aminya** [comment]
> Looks like this might be the issue [vuejs/vetur#2387](https://github.com/vuejs/vetur/issues/2387)

Yeah. seems like the same thing. Maybe we should set a project for testing instead.

**Comment by aminya** [comment]
> Normally, when I begin typing, it displays a list of possibilities, I pick the one I want, hit enter or tab and it completes it for me. That part is working fine, however for some reason it is deleting the character right in front of where the cursor started.

Maybe I understood the issue incorrectly. This seems to be the issue that was fixed in atom-languageclient

https://github.com/atom-ide-community/atom-languageclient/issues/99

**Comment by aminya** [comment]
I added a project under spec/hello-world. Open it in a new window and run `pnpm i`. You will get a lot more errors

![image](https://user-images.githubusercontent.com/16418197/101973376-382a6800-3bfd-11eb-951d-d7d21b928599.png)

**Comment by aminya** [comment]
OK. I got the datatips, outline, etc working with vls 0.5. I downgraded vls to 0.5. We need to see why 0.6 does not work.

**Comment by yoyo930021** [comment]
This is my solution and suggestion.
https://github.com/vuejs/vetur/issues/2561#issuecomment-743719403
